javascript/javascript1/week2/eventApplication.js

```diff
@@ -0,0 +1,22 @@
+
+const day = new Date();
+const weekday = new Array(7);
```

Comment:
What you did is fine...although there is a simpler way to initialize an array that you would not change in your program.
```suggestion
const weekday = {"Sunday","Monday",...}
```